content/docs/reference/glossary.md

```diff
@@ -326,6 +326,10 @@ The Pageserver uploads immutable files to cloud storage, which is the final, hig
 
 The ability to authenticate without providing a password. Neon’s [Passwordless auth](#passwordless-auth) feature supports passwordless authentication.
 
+## peak usage
+
+Peak usage is the highest amount of a resource (like storage or compute) you’ve used during the current billing period. If you go over your plan’s limit, extra charges are added in set increments. You’re charged for these extra units from the date you went over the limit, with the charges prorated for the rest of the month.
```

Comment:
```suggestion
Peak usage is the highest amount of a resource (like storage) you’ve used during the current billing period. If you go over your plan’s storage limit, extra charges are added in set increments. You’re charged for these extra units from the date you went over the limit, with the charges prorated for the rest of the month. If you go
```

Reply:
From a Billing perspective, I believe peak usage applies to storage and projects only:
![image](https://github.com/user-attachments/assets/8bbadd9f-8c09-419d-90db-7df6c6d6d7d7)